frontend/awx/access/credential-types/CredentialTypeForm.tsx

```diff
@@ -79,10 +79,10 @@ export function EditCredentialType() {
   return (
     <PageLayout>
       <PageHeader
-        title={t('Edit Credential Type')}
+        title={`${t('Edit')} ${credentialType?.name || t('Credential Type')}`}
         breadcrumbs={[
           { label: t('Credential Types'), to: getPageUrl(AwxRoute.CredentialTypes) },
-          { label: t('Edit Credential Type') },
+          { label: `${t('Edit')} ${credentialType?.name || t('Credential Type')}` },
```

Comment:
Thanks @himdel. So is the thought that for all edit areas, this should be the pattern to use?

Reply:
Any time :) I think this would not be just edits, but really any place where we're doing `translated string + translated string` or `` `${translated string} ${other string}` `` , conditionals for parts of the string, etc.

Reply:
Thanks for the clarification! For what I worked on in this PR, I think it would only apply to the "Edit" areas as the "Create" areas was just changing the casing of the words.

Reply:
Oh, if the point of the PR is something else, we can always fix the l10n later :) Didn't mean to sidetrack you into fixing l10n everywhere :)

Reply:
This PR was mainly to update the breadcrumbs/headers casing throughout AAP which included fixing the edit forms to include the resource name in the breadcrumb and headers. Would it make sense to fix the localization within this PR or at a later time?

Reply:
I'd say it's better to have separate PRs for separate tasks, but .. I won't stop you either way :)

Reply:
My concern is that if we try to fix it in a separate PR will we run into the same failing tests? And will the updates to the localization require making changes to the test files as well?

Reply:
As long as the resulting english string stays the same, it shouldn't be an issue.